app/services/reporting/agency_and_sp_report.rb

```diff
@@ -54,5 +65,13 @@ def service_providers
         ServiceProvider.where(issuer: issuers).active.external
       end
     end
+
+    def facial_match_issuers
```

Comment:
we should memoize this, or save it as a local variable in the method that calls it because the loop above will hit the database for each iteration of the loop